contrib/grpc_metrics/build.gradle

```diff
@@ -8,7 +8,8 @@ apply plugin: 'java'
 }
 
 dependencies {
-    compile project(':opencensus-api')
+    compile project(':opencensus-api'),
+            libraries.guava
```

Comment:
Could you leave a TODO here to remove the Guava dependency? Since gRPC also needs to depend on this artifact.

Reply:
I think this is covered here https://github.com/census-instrumentation/opencensus-java/issues/1113